cypress/support/ui_commands.js

```diff
@@ -311,7 +312,7 @@ Cypress.Commands.add('minDisplaySettings', () => {
     cy.get('#collapseTitle').should('be.visible', 'contain', 'Default appearance of image previews');
     cy.get('#collapseEdit').should('be.visible', 'contain', 'Edit');
 
-    cy.get('#message_displayTitle').should('be.visible', 'contain', 'Message Display');
+    cy.get('#message_displayTitle').scrollIntoView().should('be.visible', 'contain', 'Message Display');
```

Comment:
[Message display](https://build-push.internal.mattermost.com/job/mattermost-ui-testing/job/mattermost-cypress/74/artifact/mattermost-webapp/cypress/screenshots/account_settings/theme_color/custom_colors_sidebar_styles_spec.js/AS14318%20Theme%20Colors%20-%20Color%20Picker%20--%20Theme%20Display%20should%20render%20in%20min%20setting%20view%20(failed).png) is out of the window, and so it needs to scroll into view.